### PR TITLE
IW-2962 | fix banner notifications on ThemeDesigner

### PR DESCRIPTION
**Pull request by xkxd**
https://wikia-inc.atlassian.net/browse/IW-2962

@Wikia/iwing

### CODE REVIEW

File: extensions/wikia/AssetsManager/config.php [hunk 1]
```diff
@@ -875,6 +875,8 @@
 	'type' => AssetsManager::TYPE_JS,
 	'assets' => [
 		'#group_oasis_jquery',
+		'//resources/wikia/libraries/mustache/mustache.js',
+		'#group_banner_notifications_js',
```
Comment (mszabo-wikia) [inline extensions/wikia/AssetsManager/config.php]: why these changes?

File: extensions/wikia/AssetsManager/config.php [hunk 2]
```diff
@@ -875,6 +875,8 @@
 	'type' => AssetsManager::TYPE_JS,
 	'assets' => [
 		'#group_oasis_jquery',
+		'//resources/wikia/libraries/mustache/mustache.js',
```
Comment (xkxd) [inline extensions/wikia/AssetsManager/config.php]: this one is actually needed to make it work. I decided to include BannerNotifications instead of bypassing it as it's much simpler.

**Review by mszabo-wikia — APPROVED** [review]
